src/directive/show-validation.directive.ts

```diff
@@ -19,22 +19,22 @@ import { Directive, ElementRef, Renderer } from '@angular/core';
 export class ShowValidationDirective {
     constructor(el: ElementRef, renderer: Renderer) {
        let formElements = el.nativeElement.querySelectorAll('.form-group');
-       Array.prototype.forEach.call(formElements, (formGroup: any) => {
-           // TODO this needs to be properly tested
+       formElements.forEach((formGroup) =>  {
```

Comment:
@sendilkumarn did this work? coz formElements is not a proper array its array like